scalableminds/webknossos
# Disable fill tool for higher mags

This PR tackles the problem that the fill tool used in lower resolutions produces spotty segmentations.
The reason for this is that a lot of buckets get annotated at the same time and not all of them can be loaded into the memory. Therefore we restrict the usage of the tool to the first two magnifications (the same as for the trace tool).

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a volume or hybrid tracing.
- Using the fill tool should only work in the two best resolutions.
- If the fill tool is active and you zoom out, the fill tool should be disabled as soon as you zoom to a not-allowed resolution.

### Issues:
- fixes #4923

------
